kuma/wiki/jinja2/wiki/includes/wiki-notice.html

```diff
@@ -0,0 +1,7 @@
+{% set url = 'https://developer.mozilla.org/' + request.path %}
```

Comment:
I don't think so. First of all, this'll break on stage and local dev laptops. Also, `request.path` might have `...$edit` in it. What about search results? 
I suggest you move this into a jinja helper function and the initial solution could be:
```
def non_wiki_url():
    url = request.build_absolute_uri().replace('//wiki.', '//').split('$')[0]
    if ...:
    return url
```
  
```

Reply:
Perhaps, but unlikely as this is in the `document.html` template which is not used for edit pages.